content/topics/dotenv-module/_index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Introduction to Dotenv module 
+ready: true
+
+---
+
+## What is an environment variable?
+
+An [environment variable](https://en.wikipedia.org/wiki/Environment_variable) is a variable whose value is set outside the program, typically through functionality built into the operating system or [container](https://www.docker.com/resources/what-container). An environment variable is made up of a name/value pair, and any number may be created and available for reference at a point in time.
+
+For a more in-depth description of environmental variables, take a look at this: {{% contentlink "topics/linux/os-environmental-variables/" %}}
+
```

Comment:
Basic example should be above the link to the advanced content. 

I'm not even convinced that it's worth mentioning environmental variables here at all. I mean, dotenv's syntax is sortof the same but that's where the similarity ends.

It could be better to just have a link at the end for "related content" that points people at the environmental variable stuff.  

The main takeaways here should be:
- the importance of seperating config from code
- the importance of not committing your config to git
- how to achieve the above